themes/default/views/pageFormat/pageHeader.php

```diff
@@ -110,6 +110,6 @@
 			<![endif]-->
 		<!-- super fish end menus -->
 	</head>	
-	<body>
+	<body <?php if(!in_array($this->request->getController(), array('Dashboard', 'Auth'))) :?>class="withSidebar"<?php endif?>>
```

Comment:
LOL I guess that avoids doing it in javascript.

I'm just never happy am I? ;-P